server/tilesource/base.py

```diff
@@ -54,6 +59,104 @@ def __init__(self):
         self.sizeX = None
         self.sizeY = None
 
+    def _calculateWidthHeight(self, width, height, sourceWidth, sourceHeight):
```

Comment:
It actually is called ``sourceWidth, sourceHeight`` here, which is internally consistent, but doesn't match the other functions.